### DISCUSSION TITLE
Dates of findings

### DISCUSSION THREAD

**Discussion by AnotherCroft** [LimberDuck/nessus-file-analyzer#27] [Q&A]
Hi, 
Is there a way to include how old the findings are? It doesn't look like it exports out that data. It'd be nice to prioritize older findings. @damian-krawczyk 
Thank you!

**Comment by damian-krawczyk** [comment]
@AnotherCroft is it listed somewhere within single `.nessus` file? I'm affraid single `.nessus` file will not provide us this information.

We would need to somehow recognize the same target among several files given to pars.

Do you have any idea?

**Reply by AnotherCroft** [reply]
I believe you are able to compute this field using the First Found and Last Found fields. If you were to strip the datetime in the fields and negate them appropriately, are you able to append an Age column and match them according to the finding? It would be such an excellent feature because it's the last puzzle piece as we are trying to highlight old findings over 90 days. Right now, I am having to manually check your generated file against Nessus findings to line up their age.
